### DISCUSSION TITLE
How can I increase the z-index of tooltips?

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#2778] [Q&A] [answered]
How can I increase the z-index of tooltips so they appear above everything else?

I have a drawer for the left sidenav, and in some situations the tooltip appears below the drawer, which feels weird:

![Captura de pantalla 2024-01-22 a las 10 45 25](https://github.com/saadeghi/daisyui/assets/2629/82a2050d-a5d6-412e-bb12-e3841739d46d)

The drawer as `z-10`. How can I make the tooltip have a greater z index?

If I set `z-5` in the drawer I get the appearance I want - but I need `z-10` so that it works nice with the other components:

![Captura de pantalla 2024-01-22 a las 10 49 41](https://github.com/saadeghi/daisyui/assets/2629/8470e9c7-7512-4ab4-97e2-5542c8ed97cc)

If that's not possible, a workaround would be being able to include line breaks inside the tooltip text.

**Comment by saadeghi** [comment, marked as answer]
adding `before:z-[11]` (or any value) should work.

Also there are some workarounds to add new line in tooltip: https://github.com/saadeghi/daisyui/discussions/2655#discussioncomment-7912350

Let me know if you have a question.
